Core/clim-core/dialog.lisp

```diff
@@ -535,24 +535,6 @@ is called. Used to determine if any editing has been done by user")))
 
 (defparameter *no-default-cache-value* (cons nil nil))
 
-;;; Hack until more views / dialog gadgets are defined.
```

Comment:
Please explain the reasoning more clearly in the commit message.

Is the method not needed because of the fallback to `textual-dialog-view`? Then the bug is irrelevant.

Or is the fallback making things worse but unavoidable because of the bug?

Reply:
I don't remove the method `text-field-view` anymore. I only moved it in `Core/clim-core/dialog-views.lisp`. Anyway I'm not sure it works, I tried with different `:width` but I didn't see any difference.